lib/endpoint-with-defaults.ts

```diff
@@ -0,0 +1,8 @@
+export = endpointWithDefaults
+
+import merge = require('./merge')
+import parse = require('./parse')
```

Comment:
I wonder if we should replace the internal imports to ES module imports, e.g. 

```js
import merge from 'merge'
```

Any thoughts?

Reply:
We can do that. I just did a 1:1 migration to start with